docker/drone/src/test/java/org/arquillian/cube/docker/drone/VncRecorderLifecycleManagerTest.java

```diff
@@ -0,0 +1,165 @@
+package org.arquillian.cube.docker.drone;
+
+import org.arquillian.cube.spi.Cube;
+import org.arquillian.cube.spi.CubeRegistry;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.arquillian.test.spi.TestResult;
+import org.jboss.arquillian.test.spi.event.suite.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class VncRecorderLifecycleManagerTest {
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Mock
+    CubeRegistry cubeRegistry;
+
+    @Mock
+    Cube cube;
+
+    @Mock
+    After after;
+
+    @Mock
+    SeleniumContainers seleniumContainers;
+
+    @Test
+    public void shouldStartVncBeforeEachTestByDefault() {
```

Comment:
Maybe I need to rename the test. This scenario should be in another test more focused on integration test rather than unit test.